oxigen_stock_valuation_report/views/menuitems.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright NuoBiT - Frank Cespedes <fcespedes@nuobit.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl) -->
+<odoo>
+    <menuitem
+        parent="account.account_reports_management_menu"
+        action="action_oxigen_stock_valuation_wizard"
+        id="menu_detailed_inventory_valuation"
```

Comment:
Guidelines https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#security-view-and-action